<?xml version="1.0" ?>
<sdf version="1.5">
	<world name="default">

		<!-- A global light source -->
		<include>
			<uri>model://sun</uri>
		</include>

		<!-- A ground plane -->
		<include>
			<uri>model://ground_plane</uri>
		</include>


		<!--Model Blender of Stage-->

		<model name="testbed_scene">
			<static>true</static>
			<pose>6.5 37.33 0 0 0 0</pose>
			<link name="test_scene">
				<visual name="visual">
					<geometry>
						<mesh><uri>/home/luisgrvc/catkin_ws/src/mbzirc/visualization/model/stage_02_gazebo.dae</uri></mesh>
					</geometry>
				</visual>
			</link>
		</model>

        <!-- Model of Static Objects -->

        <model name="static_object_1">
            <pose>2 -2 0.05 0 0 0</pose>
            <include>
                <uri>model://object_green_cylinder_mbzirc</uri>
            </include>
        </model>

        <model name="static_object_2">
            <pose>-8 -24 0.05 0 0 0</pose>
            <include>
                <uri>model://object_green_cylinder_mbzirc</uri>
            </include>
        </model>

        <model name="static_object_3">
            <pose>-4 10 0.05 0 0 0</pose>
            <include>
                <uri>model://object_green_box_mbzirc</uri>
            </include>
        </model>

        <model name="static_object_4">
            <pose>11 -27 0.05 0 0 0</pose>
            <include>
                <uri>model://object_blue_cylinder_mbzirc</uri>
            </include>
        </model>

        <model name="static_object_5">
            <pose>-17 14 0.05 0 0 0</pose>
            <include>
                <uri>model://object_blue_cylinder_mbzirc</uri>
            </include>
        </model>

        <model name="static_object_6">
            <pose>-30 -26 0.05 0 0 0</pose>
            <include>
                <uri>model://object_blue_cylinder_mbzirc</uri>
            </include>
        </model>

        <model name="static_object_7">
            <pose>20 -5 0.05 0 0 0</pose>
            <include>
                <uri>model://object_blue_cylinder_mbzirc</uri>
            </include>
        </model>

        <model name="static_object_8">
            <pose>-36 7 0.05 0 0 0</pose>
            <include>
                <uri>model://object_red_cylinder_mbzirc</uri>
            </include>
        </model>

        <model name="static_object_9">
            <pose>6 23 0.05 0 0 0</pose>
            <include>
                <uri>model://object_red_cylinder_mbzirc</uri>
            </include>
        </model>

        <model name="static_object_10">
            <pose>-16 -14 0.05 0 0 0</pose>
            <include>
                <uri>model://object_red_cylinder_mbzirc</uri>
            </include>
        </model>

        <model name="big_object_1">
            <pose>-44 24 0.05 0 0 0</pose>
            <include>
                <uri>model://object_big_mbzirc</uri>
            </include>
        </model>

        <model name="big_object_2">
            <pose>-38 9 0.05 0 0 0</pose>
            <include>
                <uri>model://object_big_mbzirc</uri>
            </include>
        </model>

        <model name="big_object_3">
            <pose>-45 27 0.05 0 0 0</pose>
            <include>
                <uri>model://object_big_mbzirc</uri>
            </include>
        </model>
	</world>
</sdf>